app/models/cookbook.rb

```diff
@@ -258,7 +258,9 @@ def publish_version!(params)
         readme: readme.contents,
         readme_extension: readme.extension,
         changelog: changelog.contents,
-        changelog_extension: changelog.extension
+        changelog_extension: changelog.extension,
+        chef_version: metadata.chef_version,
+        ohai_version: metadata.ohai_version
```

Comment:
these should probably be plural `chef_versions` similar to `metadata.dependencies` which is plural even though the DSL method on the metadata.rb is singular "depends"